embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/SchemaType.kt

```diff
@@ -313,4 +314,26 @@ internal sealed class SchemaType(
             "emb.webview_info.tag" to tag
         ).toNonNullMap()
     }
+
+    internal class ReactNativeAction(
+        name: String,
+        outcome: String,
+        payloadSize: Int,
+        properties: Map<String?, Any?>,
```

Comment:
Can we just make this Map<String, String> since that's what we require anyway?

Reply:
We need "Any" in the public API. This place is the proper one to convert the properties to "schema attributes" to the expected format.